### PR TITLE
Add a new parameter to select a specific issue on which to remove comments on

### PR DESCRIPTION
**Pull request by delissonjunio**
As I commented on https://github.com/izhangzhihao/delete-comment/issues/30, this will be useful to clean all comments in a specific PR instead of across the repository.

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -8,16 +8,25 @@ async function run(): Promise<void> {
...
+    } else {
+      const allIssues = await octokit.paginate(
+        'GET /repos/:owner/:repo/issues?state=all',
+        {
+          owner: github.context.repo.owner,
+          repo: github.context.repo.repo
+        }
+      )
+
+      issues.push(...allIssues)
```
Comment (izhangzhihao) [inline src/main.ts:28]:
```
src/main.ts(28,19): error TS2345: Argument of type 'unknown' is not assignable to parameter of type 'string'
```

Comment (delissonjunio) [reply src/main.ts:28]: Yup, just saw that -- I'll check it out and run a test against our repo with proof. Thanks!

Comment (delissonjunio) [reply src/main.ts:28]: I've addressed the compile error now 🙂

**Comment by izhangzhihao** [comment]
Great, thanks

**Pull request merged**